README.md

```diff
@@ -456,6 +463,138 @@ $app->get('/download/tarball', [
 ], 'download-tar');
 ```
 
+### Template Variable Container
```

Comment:
That should not be in the README file and must be moved to the documentation. The informations in the README and the documentation are already different.

Reply:
@froschdesign I'll be doing a separate PR to the zend-expressive repo's documentation once this is merged and released. Unfortunately, we have the docs for this package in that repo, so we need to have something in the README here as well.

I'd like to eventually move some of the repo-specific docs out of the zend-expressive docs, but I'm not 100% sure how we can do that while keeping the information findable.

Reply:
@weierophinney 
> Unfortunately, we have the docs for this package in that repo, so we need to have something in the README here as well.

Keep it simple: Add a link to the documentation to the README file - as everywhere. In this particular case, I would also add a short note that the informations can be found in the zend-expressive docs.

Reply:
@weierophinney 
Please compare these two middleware and you will see the problem that I mean:

* https://github.com/zendframework/zend-expressive-helpers#urlhelper
* https://docs.zendframework.com/zend-expressive/v3/features/helpers/url-helper/#using-the-helper-in-middleware

Reply:
@froschdesign Noted.

I'll keep it as-is for now, and then, after the release, submit a PR to the zend-expressive repo with documentation. After that is merged, I'll send another PR here to remove the verbiage and link to the official docs.